eng/fv/ESTEC-refs.bib

```diff
@@ -0,0 +1,2005 @@
+% Use <AUTHOR>:<YEAR>:<TOPIC> for the reference labels.
+% Use the surname of the main author for AUTHOR.
+% Sort lexicographically by (YEAR, AUTHOR, TOPIC).
```

Comment:
Are all these added references actually used?

Reply:
I think all the references should be placed into `common/refs.bib`.

Reply:
That was a local copy - it wasn't/shoudn't have been in the patch set I setup.
There should be changes here somewhere to `common/refs.bib`.